presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -1808,7 +1804,7 @@ else if (deleteEmptyDirectories && !recursiveDeleteResult.isDirectoryNoLongerExi
      * <p>
      * This method will not delete anything that's neither a directory nor a file.
      *
-     * @param queryIds prefix or suffix of files that should be deleted
+     * @param queryIds               prefix or suffix of files that should be deleted
```

Comment:
accidental change?

Reply:
was automatically reformatted by intellij.  figured it was better to commit the autoformatting changes than fight with it all the time.